lldb/source/Symbol/SwiftASTContext.cpp

```diff
@@ -1448,25 +1448,36 @@ void ApplyWorkingDir(SmallString &clang_argument, StringRef cur_working_dir) {
   llvm::sys::path::append(clang_argument, cur_working_dir, rel_path);
   llvm::sys::path::remove_dots(clang_argument);
 }
+
+inline bool IsMultiArgClangFlag(const StringRef& arg) {
+  return arg == "-D" || arg == "-U" || arg == "-working-directory";
+}
+
+inline bool IsMacroDefinition(const StringRef& arg) {
+  return arg.startswith("-D") || arg.startswith("-U");
+}
+
+inline bool ShouldUnique(const StringRef& arg) {
```

Comment:
Since this is in the anonymous namespace, I think we can drop the inline keyword without any effect.